package.json

```diff
@@ -186,31 +186,35 @@
     "karma-jasmine-html-reporter": "1.5.4",
     "lint-staged": "^8.0.0",
     "ncp": "^2.0.0",
-    "ng-packagr": "12.2.1",
+    "ng-packagr": "13.0.3",
     "npm-run-all": "^4.1.5",
     "nyc": "^10.1.2",
     "ora": "^1.3.0",
+    "postcss": "^8.3.9",
```

Comment:
Out of interest, why where these imports added?

Reply:
These were added with the Nx upgrade as far as I know, but we shouldn't need them